goodtablesio/app.py

```diff
@@ -43,6 +44,8 @@
 app.register_blueprint(github)
 app.register_blueprint(s3)
 
+# Set exception logging
+Sentry(app, dsn=settings.SENTRY_DSN)
```

Comment:
Also I really like to keep local dev instances not using Sentry so perhaps we can add:

```python
if settings.SENTRY_DSN:
   ...
```